src/lib/AppRegistry.tsx

```diff
@@ -158,6 +167,34 @@ const Show: React.SFC<ShowProps> = track<ShowProps>(props => {
   return <ShowRenderer showID={showID} render={renderWithLoadProgress(Containers.Show, { showID })} />
 })
 
+interface ShowArtistsProps {
+  showID: string
+}
+const ShowArtists: React.SFC<ShowArtistsProps> = ({ showID }) => {
```

Comment:
We'll have to add analytics for those

Reply:
Makes sense – can we follow-up on that in another PR once the native changes are made?

Reply:
Yeah for sure

Reply:
Am I correct in understanding that the analytics _weren't_ there before? I'll look up the schema doc and make sure everything is accounted for.

Reply:
Yeah actually that's right, we didn't have those screens before so definitely not a blocker